3/a/aff

```diff
@@ -36,22 +36,22 @@
-{"name":"aff","version":"2.0.2","license":"Apache-2.0","location":{"githubOwner":"purescript-contrib","githubRepo":"purescript-aff"},"description":"Asynchronous effect monad","dependencies":{"arrays":">=3.1.0 <4.0.0","bifunctors":">=2.0.0 <3.0.0","console":">=2.0.0 <3.0.0","const":">=2.0.0 <3.0.0","contravariant":">=2.0.0 <3.0.0","control":">=2.0.0 <3.0.0","distributive":">=2.0.0 <3.0.0","eff":">=2.0.0 <3.0.0","either":">=2.1.0 <3.0.0","exceptions":">=2.0.0 <3.0.0","foldable-traversable":">=2.0.0 <3.0.0","functions":">=2.0.0 <3.0.0","functors":">=1.0.0 <2.0.0","identity":">=2.0.0 <3.0.0","invariant":">=2.0.0 <3.0.0","lazy":">=2.0.0 <3.0.0","maybe":">=2.0.1 <3.0.0","monoid":">=2.2.0 <3.0.0","newtype":">=1.1.0 <2.0.0","nonempty":">=3.0.0 <4.0.0","parallel":">=2.1.0 <3.0.0","partial":">=1.2.0 <2.0.0","prelude":">=2.1.0 <3.0.0","refs":">=2.0.0 <3.0.0","st":">=2.0.0 <3.0.0","tailrec":">=2.0.1 <3.0.0","transformers":">=2.0.2 <3.0.0","tuples":">=3.0.0 <4.0.0","unfoldable":">=2.0.0 <3.0.0","unsafe-coerce":">=2.0.0 <3.0.0"}}
+{"name":"aff","version":"2.0.1","license":"Apache-2.0","location":{"githubOwner":"purescript-contrib","githubRepo":"purescript-aff"},"description":"Asynchronous effect monad","dependencies":{"arrays":">=3.0.0 <4.0.0","bifunctors":">=2.0.0 <3.0.0","console":">=2.0.0 <3.0.0","const":">=2.0.0 <3.0.0","contravariant":">=2.0.0 <3.0.0","control":">=2.0.0 <3.0.0","distributive":">=2.0.0 <3.0.0","eff":">=2.0.0 <3.0.0","either":">=2.0.0 <3.0.0","exceptions":">=2.0.0 <3.0.0","foldable-traversable":">=2.0.0 <3.0.0","functions":">=2.0.0 <3.0.0","functors":">=1.0.0 <2.0.0","identity":">=2.0.0 <3.0.0","invariant":">=2.0.0 <3.0.0","lazy":">=2.0.0 <3.0.0","maybe":">=2.0.1 <3.0.0","monoid":">=2.0.0 <3.0.0","newtype":">=1.0.0 <2.0.0","nonempty":">=3.0.0 <4.0.0","parallel":">=2.0.0 <3.0.0","partial":">=1.1.2 <2.0.0","prelude":">=2.1.0 <3.0.0","refs":">=2.0.0 <3.0.0","st":">=2.0.0 <3.0.0","tailrec":">=2.0.0 <3.0.0","transformers":">=2.0.1 <3.0.0","tuples":">=2.0.0 <3.0.0","unfoldable":">=2.0.0 <3.0.0","unsafe-coerce":">=1.0.0 <3.0.0"}}
```

Comment:
Here's an example of better ranges being generated -- notice the wider acceptable ranges for `unsafe-coerce`.